app.py

```diff
@@ -16,12 +16,19 @@
 AUTOCOMPLETE_MAX_COST = 3
 AUTOCOMPLETE_SIZE = 5
 
+doc_id = 1
+
+
+
 def getAutocompleteEntries() -> dict :
     words = {}
-    with gzip.open('../wikidump/enwiki-20210820-abstract.xml.gz', 'rb') as f:
-         for _, element in etree.iterparse(f, events=('end',), tag='doc'):
-             title = element.findtext('./title')
-             words[title[11:]]={}
+    with gzip.open('../wikidump/enwiki-20210820-abstract.xml.gz', 'rb') as f: 
```

Comment:
**[Question]** How do you know that this is actually present? Remember this is in a docker container and this file is never copied in. This should be downloaded from the actual source link from wikipedia.

Reply:
ok